azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -174,7 +171,7 @@ public void testPreKilledRun() {
     final MockExecutorLoader loader = new MockExecutorLoader();
     final EventCollectorListener eventCollector = new EventCollectorListener();
     final JobRunner runner =
-        createJobRunner(1, "testJob", 1, false, loader, eventCollector);
+        createJobRunner(1, "testJob", 0, false, loader, eventCollector);
```

Comment:
This change in already enabled test `testPreKilledRun`: no reason to have sleep seconds=1 as the job is already marked as killed and skipped immediately.